pkgs/applications/graphics/conjure/default.nix

```diff
@@ -0,0 +1,71 @@
+{ fetchFromGitHub
+, gobject-introspection
+, lib
+, libadwaita
+, python3Packages
+, wrapGAppsHook
+, meson
+, ninja
+, desktop-file-utils
+, pkg-config
+, appstream-glib
+, gtk4
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "conjure";
+  version = "0.1.2";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "nate-xyz";
+    repo = "conjure";
+    rev = "v${version}";
+    hash = "sha256-qWeqUQxTTnmJt40Jm1qDTGGuSQikkurzOux8sZsmDQk=";
+  };
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+    desktop-file-utils
+    appstream-glib
+    meson
+    ninja
+    pkg-config
+    gtk4
+  ];
+
+  buildInputs = [
+    libadwaita
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    loguru
+    wand
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytest
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  meta = with lib; {
+    description = "Magically transform your images with Conjure";
```

Comment:
Remove "with Conjure" from description to not repeat the package name?

Reply:
I fixed it